CRM/Contribute/Form/Task/Invoice.php

```diff
@@ -143,6 +143,9 @@ public function buildQuickForm() {
       $this->addRule('from_email_address', ts('From Email Address is required'), 'required');
     }
 
+    $attributes = ['class' => 'huge'];
+    $this->add('text', 'cc_id', ts('CC'), $attributes);
```

Comment:
Is this just a text box for an id? 
```
    $this->addEntityRef('cc_id', ts('CC'), [
      'entity' => 'Email',
      'multiple' => TRUE,
    ]);
```
would be better

Reply:
This is just plain text field, non necessary to create contact, same like contribution page CC and BCC email.